config/config.go

```diff
@@ -247,6 +71,9 @@ func Validate(conf Config) error {
 		if conf.Matcher.ConnString == "" {
 			return fmt.Errorf("matcher mode requires a database connection string")
 		}
+		if conf.Notifier.ConnString == "" {
+			return fmt.Errorf("notifier mode requires a database connection string")
+		}
```

Comment:
This would be odd one-off thing to do for notifier. Right now the config requires a db string be present for each service so that it works the same if you decide not to run in combo-mode.

I do see what you're getting at here, we only need to "find" one db string for combo mode, but it combo mode can also be pointed to different databases, its probably better to be explicit in regards to database strings since it works the same for any mode.

Reply:
I forgot the notifier has its own DB; this suggestion sucks.